platform_plugin_turnitin/static/js/src/turnitin.js

```diff
@@ -0,0 +1,273 @@
+              updateTrafficLightState('1', 'ERROR');
+              alert(`ERROR: ${response.status} - ${response.message}`);
+          } else {
+              updateTrafficLightState('1', response['status']);
+          }
+            
+        },
+        error: function() {
+            alert('Error getting report status.');
+        }
+    });
+  });
+
+
+
+  $('#generateReportBtn1', element).click(function(event) {
+      event.preventDefault();
+      var handlerUrl = runtime.handlerUrl(element, 'generate_similarity_report');
+
+      $.ajax({
+          type: "POST",
+          url: handlerUrl,
+          data: JSON.stringify({"hello": "world"}),
+          success: function(response) {
+              alert('Successfully scheduled similarity report generation.');
+          },
+          error: function() {
+              alert('Error in similarity report generation.');
+          }
+      });
+  });
+
+
+
+
+  $('#refreshBtn2', element).click(function(event) {
+      event.preventDefault();
+      var handlerUrl = runtime.handlerUrl(element, 'get_similarity_report_status');
+
+      $.ajax({
+          type: "POST",
+          url: handlerUrl,
+          data: JSON.stringify({"hello": "world"}),
+          success: function(response) {
+            if(response.success === false && response.status === 404) {
+                updateTrafficLightState('2', 'ERROR');
+                alert(`ERROR: ${response.status} - ${response.message}`);
+            } else {
+                updateTrafficLightState('2', response['status']);
+            }
+              
+          },
+          error: function() {
+              alert('Error getting report status.');
+          }
+      });
+  });
+
+
+  $('#generateReportBtn2', element).click(function(event) {
+      event.preventDefault();
+      var handlerUrl = runtime.handlerUrl(element, 'create_similarity_viewer');
+
+      $.ajax({
+          type: "POST",
+          url: handlerUrl,
+          data: JSON.stringify({"hello": "world"}),
+          success: function(response) {
+              alert('Redirecting...');
+              openInNewTab(response['viewer_url']);
+          },
+          error: function() {
+              alert('Error getting Viewer URL.');
+          }
+      });
+  });
+
+
+  function openInNewTab(url) {
+      window.open(url, '_blank');
+  }
+
+
+  function updateTrafficLightState(semaphoreNumber, state) {
+      console.log(state);
+      const redLight = document.getElementById(`redLight${semaphoreNumber}`);
+      const yellowLight = document.getElementById(`yellowLight${semaphoreNumber}`);
+      const greenLight = document.getElementById(`greenLight${semaphoreNumber}`);
+      const generateReportBtn = document.getElementById(`generateReportBtn${semaphoreNumber}`);
+      const statusText = document.getElementById(`statusText${semaphoreNumber}`);
+      const refreshBtn2 = document.getElementById(`refreshBtn2`);
```

Comment:
Same in this [comment](https://github.com/eduNEXT/platform-plugin-turnitin/pull/4/files#r1347738955) for these constatants